torchtitan/models/llama/__init__.py

```diff
@@ -48,4 +48,13 @@
         multiple_of=4096,
         rope_theta=500000,
     ),
+    "405B": ModelArgs(
+        dim=16384,
+        n_layers=126,
+        n_heads=128,
+        n_kv_heads=16,
```

Comment:
🤔 I think the `n_kv_heads` should be 8 instead of 16?

Reply:
There are some confusions around this. Turns out that we were misled by the errors from HuggingFace.